word/models.py

```diff
@@ -27,7 +27,7 @@ class Description(models.Model):
                                  on_delete=models.SET_NULL, null=True)
 
     def __str__(self):
-        return self.short
+        return self.short + ": " + self.extended

```

Comment:
It is generally better to use string formatting. Since we are using Python 3.6 you can do this:
`f"{self.short}: {self.extended}"`

Reply:
thanks